vizro-core/src/vizro/models/_components/form/_text_area.py

```diff
@@ -36,17 +48,25 @@ class TextArea(VizroBaseModel):
 
     @property
     def _action_outputs(self) -> dict[str, _IdProperty]:
-        return {"__default__": f"{self.id}.value"}
+        return {
+            "__default__": f"{self.id}.value",
+            "title": f"{self.id}_title.children",
+            **({"description": f"{self.description.id}.children"} if self.description else {}),
+        }
 
     @property
     def _action_inputs(self) -> dict[str, _IdProperty]:
         return {"__default__": f"{self.id}.value"}
 
     @_log_call
     def build(self):
+        description = self.description.build().children if self.description else [None]
+
         return html.Div(
             children=[
-                dbc.Label(self.title, html_for=self.id) if self.title else None,
+                dbc.Label(children=[html.Div(self.title, id=f"{self.id}_title"), *description], html_for=self.id)
```

Comment:
Comment applies in many places. 
```suggestion
                dbc.Label(children=[html.Span(self.title, id=f"{self.id}_title"), *description], html_for=self.id)
```

Reply:
Should I do it for all 13 occurrences 😆? So: graph, ag-grid, table, user-input, text-area, checklist, dropdown, radio-items, slider, range-slider, date-picker, tabs, container, and _dashboard(it displays page’s title).

Reply:
Done in:
- code -> https://github.com/mckinsey/vizro/pull/1178/commits/1e311aefc8391e10fe42b0930ba99aef41809e49
- tests -> https://github.com/mckinsey/vizro/pull/1178/commits/51d88515f9fbcc96025d32ff3d3be7ff7228553d

Reply:
Yes 😀 Assuming the results of the screenshot tests look good.

Reason for the change is that div is block-level but span is inline: https://en.wikipedia.org/wiki/Div_and_span.